documentation/asciidoc/computers/configuration/host-wireless-network.adoc

```diff
@@ -0,0 +1,165 @@
+                                     +---+ 10.10.0.2   |          +- Laptop ----+
+                                     |   |     WLAN AP +-)))  (((-+ WLAN Client |
+                                     |   | 192.168.1.1 |          | 192.168.1.2 |
+                                     |   +-------------+          +-------------+
+                 +- Router ----+     |
+                 | Firewall    |     |   +- PC#2 ------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 10.10.0.3   |
+                 |   10.10.0.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 10.10.0.4   |
+                                         +-------------+
+----
+TODO: Switch to an image version of this ascii art!
+////
+
+With this network configuration, wireless clients can all communicate with each other through the Raspberry Pi router. However, clients on the wireless network cannot directly interact with clients on the wired network other than the Raspberry Pi; wireless clients exist in a private network separate from the network that serves wired clients.
+
+NOTE: The Raspberry Pi 4, Raspberry Pi 3, and Raspberry Pi Zero W can host a wireless network using the built-in wireless module. Raspberry Pi models that lack a built-in module support this functionality using a separate wireless dongle.
+
+=== Prerequisites
+
+To host a wireless network with your Raspberry Pi, you will need the following:
+
+* a Raspberry Pi with a wireless module (built-in or separate)
+* administrative access to your Raspberry Pi
+* a wireless client, such as a laptop or smartphone, to test your new wireless network
+
+=== Setup
+
+You can configure a hosted wireless network on your Raspberry Pi through the Network Manager app on the desktop or on the command line with `nmcli`.
+
+==== Graphical desktop
+
+To create a hosted wireless network using the Network Manager app on the desktop, complete the following steps:
+
+1. Click the network icon in the system tray. Select *Advanced Options* > *Create Wi-Fi Hotspot*.
++
+--
+image::images/create-hotspot-network-menu.png[Opening the Hotspot creation menu from the desktop]
+--
+2. Enter a network name.
++
+--
+image::images/create-hotspot-dialog.png[The Network Manager Hotspot creation menu]
+--
+3. From the *Wi-Fi Security* dropdown, select *WPA and WPA2 Personal*.
+4. Enter a password. WPA encryption requires a password at least 8 characters long.
+5. Click the *Create* button to begin hosting a wireless network from your Raspberry Pi.
+
+To disable the network:
+
+1. Click the network icon in the system tray. Select *Turn Off Wireless LAN*.
+2. Click the network icon in the system tray. Select *Turn On Wireless LAN*.
+
+The wireless module should automatically connect to your preferred wireless network when you complete the second step. If it doesn't, pick a wireless network from the dropdown.
```

Comment:
Huh, this section doesn't seem to make sense?

Reply:
What specifically doesn't make sense?

Reply:
Perhaps it was just me misunderstanding things, but it seems to be saying "To disable the network... turn it off and on again" ??

Reply:
Might also be worth clarifying that when you "turn it on again" the hotspot network will have been disabled? I think that might have been what was confusing me.

Reply:
Does this mean that the hotspot network doesn't automatically survive a reboot? Or does NetworkManager treat a reboot event differently to disabling and then re-enabling the WiFi?

Reply:
Good question. Basically:

* network manager always connects to the highest priority connection on launch (including the hotspot)
* the UI doesn't include any way to disable the hotspot and switch back to wireless client mode

It is tempting to remove the UI instructions entirely here and point users to the CLI method instead. Increasingly I suspect it might be the *only* viable option.

Reply:
Ugh. Apologies for opening the hornet's nest, but I'll leave you and Alasdair to thrash out the best solution :wink:

Reply:
For this, considering, I think we should dump the GUI instructions and revert to CLI only.